package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "top-video-player",
+  "version": "",
+  "description": "",
+  "author": "",
+  "main": "index.js",
+  "license": "",
+  "engines": {
+    "node": ">=6.13.0"
+  },
+  "scripts": {
+    "seed": "node server/seeder.js",
+    "build": "webpack --config webpack.config.js",
+    "nodemon": "nodemon server/index.js"
+  },
+  "dependencies": {
+    "axios": "^0.19.0",
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "mongoose": "^5.7.4",
+    "mongoose-auto-increment": "^5.0.1",
+    "mongoose-data-seed": "^2.1.4",
+    "mongoose-seed": "^0.6.0",
```

Comment:
you may want to clean these out of your dependencies since you're not using them :)

Reply:
Good looks! Thanks.